### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by al-fisher**
### NOTE: Depends on 
https://github.com/BHoM/documentation/pull/92

as part of restructuring documentation navigation.

### CODE REVIEW

**Comment by al-fisher** [comment]
@BHoMBot check required

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This repository can be used to create brand new BHoM Toolkits :rocket:
 - Start developing! :rocket:
 
 ## Configure development environment
-Learn how to [set up your development environment for BHoM here](https://bhom.xyz/documentation/Contributing/Getting-started-for-developers/). 
+Learn how to [set up your development environment for BHoM here](https://bhom.xyz/documentation/Guides-and-Tutorials/Coding-with-BHoM/). 
 
 ## Implement the Toolkits
-See the instructions in [the documentation page on the BHoM_Toolkit](https://bhom.xyz/documentation/Basics/The-BHoM-Toolkit).
+See the instructions for in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```
Comment (FraserGreenroyd) [inline README.md]: Is this missing a word? Or is `for` supposed to be removed? I'm guessing it's meant to be something like:
```suggestion
See the instructions for creating a toolkit in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```

Though that seems verbose, so maybe:

```suggestion
See the instructions in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```

Comment (FraserGreenroyd) [reply README.md]: It is also late so it could just be my eyes! 👀

Comment (al-fisher) [reply README.md]: Good catch!! tbh needed a rewrite 😸 
See new commit!

**Review by FraserGreenroyd — APPROVED** [review]
Changes align with documentation changes, approving so this will be ready for merge once documentation goes in.

**Pull request merged**